core/scss/layout/three-column/_blastila-right.scss

```diff
@@ -0,0 +1,31 @@
+@charset "UTF-8";
+
+//
+// Blastila Right - Variant
+//
+.layout-blastila--right {
+
+  > div {
+
+    > aside {
+      // First column, for 1 column, but down two columns.
+      @include grid-item-spans(3, span 3, 1, span 2);
```

Comment:
```suggestion
      @include grid-item-spans(3, span 1, 1, span 2);
```

Reply:
Looks like a small typo for the aside span. Changing from 3 to 1 would make it mirror the default version of Blastila.

Reply:
Good catch! I've merge this change in.